app/views/movies/index.html.erb

```diff
@@ -0,0 +1,88 @@
+<div class="homepage">
+
+    <h2 class="homepage-header">IMDB </h2>
+    <% if @movies_list.length >0 %>
+        <h4 class="homepage-subheader"> Recent Reviews </h4>
+        <div class="flexslider">
+            <ul class="slides">
+            <% @movies_list.each do |movie| %>
+
+            <% next if movie[:Response] == "False" %>
+
+            <li>
+            <div class="home-movie inline-block">
+              <div class="home-movie-poster"> <img class="movie-image" src="<%= movie[:Poster] %>"> </div>
+              <div class="overlay white-overlay">
+                <div class="overlay-title">
+                    <%=link_to movie[:Title], "/movies/#{movie[:imdbID]}", class: "homepage-link" %>
+                </div>
+                <div class="overlay-stars">
+                    <%= Review.for_movie(movie[:imdbID])[-1].stars %>
```

Comment:
Let's not reach into an array in the view. Have the model deliver to the view exactly what it needs.